DatadogRUM/Sources/DataModels/RUMDataModels.swift

```diff
@@ -4334,4 +4335,4 @@ public struct RUMTelemetryOperatingSystem: Codable {
     }
 }
 
-// Generated from https://github.com/DataDog/rum-events-format/tree/ae8c30a094339995e234fd55831ade0999bf0612
+// Generated from https://github.com/DataDog/rum-events-format/tree/c853be6db33125c8767ae563d2af47b92636c4e1
```

Comment:
note/ To not forget updating it after https://github.com/DataDog/rum-events-format/pull/215 is merged

Reply:
Yes, regened models as dependency PR is merged.